pkg/runners/builtin_runner/range_check.go

```diff
@@ -0,0 +1,69 @@
+package builtinrunner
+
+import (
+	"errors"
+	"math"
+
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+)
+
+const INNER_RC_BOUND_SHIFT = 16
+const INNER_RC_BOUND_MASK = math.MaxUint16
+const CELLS_PER_RANGE_CHECK = 1
+
+const N_PARTS = 8
+
+type RangeCheckBuiltinRunner struct {
+	base     memory.Relocatable
+	included bool
+}
+
+func NewRangeCheckBuiltinRunner(ratio *uint32, nParts uint32, included bool) *RangeCheckBuiltinRunner {
```

Comment:
```suggestion
func NewRangeCheckBuiltinRunner(included bool) *RangeCheckBuiltinRunner {
```
Maybe we can remove these args since they aren't used

Reply:
They are not used for now, but in a future we will

Reply:
Done, removed!